deploy/firstboot/centos7-kvm.sh

```diff
@@ -93,3 +93,8 @@ STP=yes" > /etc/sysconfig/network-scripts/ifcfg-cloudbr1
 # Reboot
 reboot
 
+# Keep the script running until the reboot happens
+while :
+do
+  # loop infinitely
```

Comment:
So it is needed to keep the shell that does the reboot running. Hope it is run as root